### DISCUSSION TITLE
className을 어떤 방식으로 통일하면 좋을까요?

### DISCUSSION THREAD

**Discussion by codeisneverodd** [JCTG-JavaScript-Coding-Test-Group/SolutionBank#21] [0. 개발]
아래의 사진처럼 두가지 경우로 쓰이고 있는데 하나로 통일할 필요성이 있어보입니다! 의견 남겨주세요
![image](https://user-images.githubusercontent.com/54318460/172568153-ce6c111b-87c6-4f64-a93c-c0802260a798.png)

![image](https://user-images.githubusercontent.com/54318460/172568123-f66a8dc1-ec3b-45cf-b0f4-a736a0903b94.png)

**Comment by chaerin-dev** [comment]
지난번에 경현님께서 말씀해주신대로 IDE에서 수정시 위 방식이 더 편리하다고 생각합니다.

**Comment by yongchanson** [comment]
카멜표기법으로 수정하는게 좋다고 생각합니다.
